src/buckets/components/createBucketForm/CreateBucketForm.tsx

```diff
@@ -19,16 +19,21 @@ import {
   DEFAULT_RULES,
 } from 'src/buckets/reducers/createBucket'
 import {AppState, Bucket, RetentionRule} from 'src/types'
+import {event as influxEvent} from '../../../cloud/utils/reporting'
```

Comment:
is `event` already used here? any reason not to just leave this as `event`?

also, can you change this import to be absolute, not relative please?

Reply:
when i put it in as 'event' i got errors.  (there are form events and such on the page).  I can try again.

will find an absolute import and copy it.

Reply:
fixed; the method that event(<event string>) gets called in has an argument/parameter called 'event'.  so fixed now.